Sources/DataCache.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  DataCache.swift
+//  Contentful
+//
+//  Created by JP Wright on 31.07.17.
+//  Copyright © 2017 Contentful GmbH. All rights reserved.
+//
+
+import Foundation
+
+
+/// Implemented using `NSCache`
+internal class DataCache {
+
+    static let cacheKeyDelimiter = "_"
+
+    public static func cacheKey(for resource: EntryModellable) -> String {
+        let delimeter = DataCache.cacheKeyDelimiter
+
+        let cacheKey =  resource.id + delimeter + "entry" + delimeter + resource.localeCode
+        return cacheKey
+    }
+
+    public static func cacheKey(for resource: LocalizableResource) -> String {
+        let delimeter = "_"
```

Comment:
hmm, I am seeing double here. Do you mind using the `let constant` of the `DataCache` here?